models/ios/prefix_lists/ios_prefix_lists.yaml

```diff
@@ -0,0 +1,86 @@
+module: ios_prefix_lists
+short_description: Prefix Lists resource module
+description:
+  - This module configures and manages the attributes of prefix list on Cisco IOS.
+version_added: 2.2.0
+author: Sumit Jaiswal (@justjais)
+notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  - This module works with connection C(network_cli).
+    See L(IOS Platform Options,../network/user_guide/platform_ios.html)
+options:
+  config:
+    description: A list of configurations for Prefix lists.
+    type: list
+    elements: dict
+    suboptions:
+      afi:
+        description:
+          - The Address Family Indicator (AFI) for the  prefix list.
+        type: str
+        choices: ['ipv4', 'ipv6']
+      prefix_lists:
+        description: List of Prefix-lists.
+        type: list
+        elements: dict
+        suboptions:
+          name:
+            description: Name of a prefix-list
+            type: str
+          params:
```

Comment:
To keep this consistent with `route_maps`, can we have this as `entries` because these also have sequence numbers?

Reply:
+1